index.js

```diff
@@ -1,6 +1,7 @@
 const mongoose = require('mongoose');
 const app = require('./src/app');
 
-mongoose.connect(process.env.DATABASE_CONN, { useNewUrlParser: true }, () => {
+mongoose.connect(process.env.DATABASE_CONN, { useNewUrlParser: true }, (err) => {
+  console.log(err);
   app.listen(3000);
```

Comment:
Pro Tip: app.listen also take a callback function. I use it to remind myself which port the app is on:

```
const port = process.env.PORT

mongoose.connect(process.env.DATABASE_CONN, { useNewUrlParser: true }, (err) => {
  console.log(err);
  app.listen(port, () => {
     console.log(`App is running on port ${port}`)
  });
```

Reply:
This makes working with hosting services like Heroku a bit easier.

Reply:
```suggestion
  app.listen(3000);
```